proposal-templates/0000-swiftpm-template.md

```diff
@@ -7,7 +7,7 @@
 
 *During the review process, add the following fields as needed:*
 
-* Implementation: [apple/swift-package-manager#NNNNN](https://github.com/apple/swift-package-manager/pull/NNNNN)
```

Comment:
This change should be part of a separate PR so we can merge it without waiting for this proposal's review. :)